__init__.py

```diff
@@ -674,6 +673,11 @@ def _get_alarm_matches(
             )
 
             if reply:
+                # if user wants to bail, bail!
+                if self.voc_match(utt, "Terminate"):
```

Comment:
I wonder if we should use the global `mycroft-core/mycroft/res/text/lang-code/cancel.voc`?
Note you shouldn't have to do anything special, just `voc_match(utt, "cancel") and if there's no `cancel.voc` in the Skill it will look in the above file.

Would need to expand the vocab in that file but would mean a more consistent experience for cancelling things across all Skills.

Reply:
That's an excellent idea. Will update accordingly.